data/hai/unfettered hunger.txt

```diff
@@ -0,0 +1,395 @@
+		add fleet "Emergency Sympathizer Food" 4000
+	system "Ula Mon"
+		add fleet "Emergency Sympathizer Food" 4000
+	system "Mei Yohn"
+		add fleet "Emergency Sympathizer Food" 2000
+	system "Zuba Zub"
+		add fleet "Emergency Sympathizer Food" 1500
+	system "Wah Ki"
+		add fleet "Emergency Sympathizer Food" 800
+	
+	# increase the price of food by 100
+	system "Wah Yoot"
+		add fleet "Emergency Symathizer Food" 1200
+		trade Food 698
+	system "Ehma Ti"
+		add fleet "Emergency Sympathizer Food" 2500
+		trade Food 670
+	system "Hi Yahr"
+		add fleet "Emergency Sympathizer Food" 2500
+		trade Food 640
+
+	system "Chy'chra"
+		add fleet "Unfettered Raid" 5000
+	system "Prakacha'a"
+		add fleet "Unfettered Raid" 5000
+	system "Ehma Ti"
+		add fleet "Unfettered Raid" 2500
+	system "Wah Ki"
+		add fleet "Large Unfettered" 1200
+
+
+event "unfettered: need for food stops"
+	clear "unfettered: hunger stops"
+	clear "event: unfettered: need for food"
+	clear "unfettered: hunger"
+	system "Wah Ki"
+		remove fleet "Emergency Sympathizer Food"
+	system "Bore Fah"
+		remove fleet "Emergency Sympathizer Food"
+	system "Bote Asu"
+		remove fleet "Emergency Sympathizer Food"
+	system "Ula Mon"
+		remove fleet "Emergency Sympathizer Food"
+	system "Ya Hai"
+		remove fleet "Emergency Sympathizer Food"
+	system "Io Lowe"
+		remove fleet "Emergency Sympathizer Food"
+	system "Zuba Zub"
+		remove fleet "Emergency Sympathizer Food"
+	system "Wah Oh"
+		remove fleet "Emergency Sympathizer Food"
+	system "Due Yoot"
+		remove fleet "Emergency Sympathizer Food"
+	system "Io Mann"
+		remove fleet "Emergency Sympathizer Food"
+	system "Rati Cal"
+		remove fleet "Emergency Sympathizer Food"
+	
+	system "Wah Yoot"
+		remove fleet "Emergency Symathizer Food"
+		trade Food 598
+	system "Ehma Ti"
+		remove fleet "Emergency Sympathizer Food"
+		trade Food 570
+	system "Hi Yahr"
+		remove fleet "Emergency Sympathizer Food"
+		trade Food 540
+
+	system "Chy'chra"
+		remove fleet "Unfettered Raid"
+	system "Prakacha'a"
+		remove fleet "Unfettered Raid"
+	system "Ehma Ti"
+		remove fleet "Unfettered Raid"
+	system "Wah Ki"
+		remove fleet "Large Unfettered"
+		remove fleet "Large Unfettered"
+		add fleet "Large Unfettered" 550
+
+
+fleet "Emergency Sympathizer Food"
+	government "Hai Merchant (Sympathizers)"
+	names "hai"
+	commodities "food"
+	personality
+		confusion 25
+		timid frugal appeasing
+	variant 3
+		"Shield Beetle (Tracker)"
+		"Water Bug" 3
+	variant 1
+		"Shield Beetle (Tracker)"
+		"Centipede" 3
+	variant 2
+		"Water Bug" 3
+		"Centipede" 2
+	variant 1
+		"Centipede" 3
+		"Centipede (Pulse)" 2
+	variant 3
+		"Centipede" 2
+		"Lightning Bug" 3
+		"Water Bug"
+	variant 2
+		"Aphid (Armed)" 2
+		"Lightning Bug"
+		"Centipede"
+	variant 1
+		"Geocoris (Tracker)"
+		"Shield Beetle"
+		"Aphid" 2
+		"Aphid (Armed)"
+	variant 1
+		"Centipede"
+		"Geocoris (Tracker)"
+		"Lightning Bug"
+		"Water Bug"
+		"Grasshopper (Tracker)"
+
+
+
+mission "Unfettered: Hunger Stories 1"
```

Comment:
Are you sure FL can't have a translator or something when doing this mission?

Reply:
They could indeed, and that's what I said to @williaji they will take care of it when they have the time